File: benchmark/python/strrev.py
import time
s = "a" * 50000
start = time.time() * 1000
rev = ""
for j in range(len(s) - 1, -1, -1):
    rev += s[j]
end = time.time() * 1000
print(f"Len: {len(rev)}")
print(f"Time: {end - start:.0f}ms")

File: benchmark/perl/strrev.pl
use Time::HiRes qw(time);
my $s = "a" x 50000;
my $start = time();
my $rev = "";
for my $j (reverse 0..length($s)-1) { $rev .= substr($s, $j, 1); }
my $end = time();
my $ms = ($end - $start) * 1000;
print "Len: " . length($rev) . "
";
printf "Time: %.0fms
", $ms;
